File: dotnet/src/adapter/driving/WellKnownTestResponseCodeController.cs
using System.Dynamic;
using BackendExample.Utility;
using Microsoft.AspNetCore.Mvc;

namespace BackendExample.Adapter.Driving
{
    /// <summary>
    /// 测试响应状态码 Well Known Controller
    /// </summary>
    public class WellKnownTestResponseCodeController : ControllerBase
    {
        /// <summary>
        /// 测试响应 500
        /// </summary>
        [HttpGet("/.well-known/test/response/code/500")]
        [Produces("text/plain")]
        public IActionResult testResponseCode500()
        {
            return StatusCode(500, "测试500错误");
        }

    }
}

File: python/src/adapter/driving/well_known_test_response_code_controller.py
import logging

from fastapi import APIRouter, Response

logger = logging.getLogger(__name__)
router = APIRouter()


@router.get("/.well-known/test/response/code/500")
def test_response_code_500():
    """测试响应 500"""
    return Response(status_code=500, content="测试500错误", media_type="text/plain; charset=utf-8")

